classes/hunter.lua

```diff
@@ -29,6 +30,10 @@ local function registerClass(self)
     -- Counterattack, registered as both aura and counter, but only used as counter
     self:RegisterAura("counterattack", 0, counterattack, nil, "", 0, 0, 0, 0, false, { (GetSpellInfo(counterattack)) });
     self:RegisterCounter("counterattack"); -- Must match name from above call
+
+    -- Kill Shot, execute based on 20% and below
+    self:RegisterAura("killShot", 0, killShot, nil, "", 0, 0, 0, 0, false, { {GetSpellInfo(killShot)} });

```

Comment:
The name should be either `"killshot"` or `"kill_shot"`, not `"killShot"`.
I would suggest `"kill_shot"`, given the spell name is Kill Shot (two words).

Reply:
Fixed